package.json

```diff
@@ -58,6 +58,7 @@
     "@lingui/react": "^4.5.0",
     "@mattermost/react-native-paste-input": "^0.6.4",
     "@miblanchard/react-native-slider": "^2.3.1",
+    "@radix-ui/react-dropdown-menu": "^2.0.6",
```

Comment:
Zeego relies on `2.0.1`, which was the last breaking release. Patches since then were all trivial, and I've observed no issues with existing dropdowns on web.